src/components/arm/Arm.ts

```diff
@@ -0,0 +1,60 @@
+import type { Pose, WorldState } from '../../gen/common/v1/common_pb.esm';
+
+import type { JointPositions } from '../../gen/component/arm/v1/arm_pb.esm';
+
+import type { Extra } from '../../types';
+
+export interface Arm {
+  /**
+   * Get the position of the end of the arm expressed as a pose
+   *
+   * @returns Pose
```

Comment:
```suggestion
```

Reply:
we can omit `@returns` unless we add details beyond what's in the main doc description